relay-server/src/services/store.rs

```diff
@@ -1522,7 +1592,8 @@ struct LogKafkaMessage<'a> {
     retention_days: u16,
     #[serde(default)]
     received: u64,
-    body: &'a RawValue,
+    #[serde(default)]
+    body: Cow<'a, str>,
```

Comment:
I'm not sure the `LogKafkaMessage` needs `timestamp_nanos` `body` etc. and all these properties now since everything should be in attributes, or added into attributes (eg. during otel -> ourlog conversion), they were all added into `other` to keep supporting the existing consumer.

Reply:
Yes, this was removed.